SimpleAkkaDotNetCluster/SimpleClusterCaller/Program.cs

```diff
@@ -18,46 +16,28 @@
     var _baseUrl2 = "akka.tcp://ClusterSystem@localhost:2552";
 
     var thisPort = 3000;
-    var baseLocation = Assembly.GetAssembly(typeof(Program));
-    var dirInfo = new DirectoryInfo(baseLocation.Location);
-    var hoconFilePath = dirInfo.Parent + "\\akka-hocon.conf";
-    Console.WriteLine(hoconFilePath);
-    var hoconFile = XElement.Parse(File.ReadAllText(hoconFilePath));
-
-    var config = ConfigurationFactory.ParseString(hoconFile.Descendants("hocon").Single().Value);
+    var hoconFile = File.ReadAllText("akka-hocon.conf");
+    var config = ConfigurationFactory.ParseString(hoconFile);
 
     var clusterNodeConfig =
                         ConfigurationFactory.ParseString("akka.remote.dot-netty.tcp.port=" + thisPort)
                             .WithFallback(config);
 
     //create an Akka system
     Console.WriteLine("Create actor system");
-    var actorSystem = ActorSystem.Create("ClusterSystem", clusterNodeConfig);
+    var actorSystem = ActorSystem.Create("ClusterClientSystem", clusterNodeConfig);
```

Comment:
The `ActorSystem` node that hosts the `ClusterClient` __must not__ be a part of the cluster, it has to belong to a different `ActorSystem` name.

Reply:
I stand corrected, @Aaronontheweb is correct.